lua/nvim-treesitter/query.lua

```diff
@@ -105,7 +105,7 @@ do
   ---@param query_name string
   function M.get_query(lang, query_name)
     if cache[lang][query_name] == nil then
-      cache[lang][query_name] = ts.get_query(lang, query_name)
+      cache[lang][query_name] = ts.get(lang, query_name)
```

Comment:
if the file is correct, `ts` is `nvim-treesitter.compat`, and `ts.get` is not a thing, only `get_query` exists though, no?

Reply:
That's correct, however if you look at the fully qualified call in